Makefile

```diff
@@ -159,8 +167,8 @@ run_test    = printf "Executing:   $1:  "; \
              if [ $$OUT -eq 0 ]; then \
                  echo "OK"; \
              else \
-                 rm $1; \
-                 exit 1; \
+                  rm $1; \
+                  exit 1; \
```

Comment:
Minor nit: not sure if intended but by adding space in front of these two lines makes them no longer aligned with the `echo "OK"; \ ` instruction above. Was this change intended?